lib/buckets/common.js

```diff
@@ -53,8 +58,21 @@ var resemblesIpRegexStr = format('^%s\.%s\.%s\.%s$', threeDigitRegexStr,
 var resemblesIpRegex = new RegExp(resemblesIpRegexStr);
 
 function isValidBucketName(name) {
-    return bucketRegex.test(name) && !resemblesIpRegex.test(name) &&
-        name.length >= 3 && name.length <= 63;
+    assert.string(name, 'name');
```

Comment:
I know you didn't add the sizes here, but it would be good to have some docs on these values. I assume they are coming from S3: https://docs.aws.amazon.com/AmazonS3/latest/dev/BucketRestrictions.html#bucketnamingrules and DNS requirements.

Reply:
There are based on S3, but are documented more thoroughly here: https://github.com/joyent/manta-buckets-api/blob/master/docs/index.md#restrictions